src/web-ui/src/public/Welcome.vue

```diff
@@ -0,0 +1,127 @@
+          personalized product recommendations a shopper would receive based on historical and real time click-through
+          behavior.
+        </p>
+
+        <div class="mt-2 d-flex flex-column flex-lg-row align-items-lg-center">
+          <div class="mb-2 mb-lg-0">AWS Services Included:</div>
+
+          <ul
+            class="service-list mb-0 pl-0 flex-grow-1 d-flex flex-column flex-lg-row justify-content-lg-around font-weight-bold"
+          >
+            <li v-for="service in services" :key="service.text" class="mb-2 mb-lg-0">
+              <ServiceBadge :text="service.text" :src="service.src"></ServiceBadge>
+            </li>
+          </ul>
+        </div>
+
+        <!-- make both links to go /auth until we have dedicated sign in and create account pages -->
+        <div class="auth-cta my-5 d-flex flex-column flex-sm-row justify-content-end align-items-center">
+          <div class="login-cta d-flex align-items-center">
+            *Have an account?<router-link to="/auth" class="sign-in ml-3 btn btn-link">Sign in</router-link>
+          </div>
+          <router-link to="/auth" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary">Create an account</router-link>
+        </div>
+      </header>
+
+      <section class="info py-3 px-3 py-md-4 px-md-5">
+        <p>
+          *To keep the click-through data fresh, this demo resets every day at 11 pm EST. Accounts created more than 24
+          hours prior would be deleted as well. If your account has been deleted, go ahead and create a new one.
+        </p>
+        <p class="mb-0">
+          This project also comes with a collection of workshops that are designed to walk a technical audience, either
+          in a group setting or self-paced as an individual, through adding enhanced functionality to the Retail Demo
+          Store. There are workshops for adding search, personalization, experimentation framework, user messaging, and
+          more. For an overview of the Retail Demo Store, its architecture, and workshops, please see the
+          <a
+            href="https://github.com/aws-samples/retail-demo-store/tree/master/workshop"
+            target="_blank"
+            rel="noopener noreferrer"
+            class="workshop-link"
+            >workshop welcome notebook page</a
+          >.
+        </p>
+      </section>
+    </div>
+  </Layout>
+</template>
+
+<script>
+import Layout from './components/Layout';
+import ServiceBadge from './components/ServiceBadge';
+
+export default {
+  name: 'Welcome',
+  components: {
+    Layout,
+    ServiceBadge,
+  },
+  data() {
+    return {
+      services: [
+        {
+          text: 'Amazon Personalize',
+          src: '/personalize.svg',
+        },
+        {
+          text: 'Amazon Pinpoint',
+          src: '/pinpoint.svg',
+        },
+        {
+          text: 'Amazon Lex',
+          src: '/lex.svg',
+        },
+      ],
```

Comment:
There are many more services than these used in the project. I know that the intent is to highlight the services that are related to customer experience but maybe we need to set that up better. I'm sure we'll be making some wording/marketing copy sweeps across all of these changes before it's all done so leaving this as a thought for now.

Reply:
That's fair. We'll be sure to revisit this issue before our deliverable date.